mobile/components/views/Attractions/Search/Search.tsx

```diff
@@ -1,122 +1,110 @@
-import React, { useState, useEffect } from 'react';
-import { useNavigation } from '@react-navigation/native';
-import { View, Button, FlatList, TouchableOpacity, Image, Alert, Text } from 'react-native';
-import FiltersModal from './FiltersModal';
-import { attractionsExamples } from './api/apiMock';
-import AttractionModal from './AttractionModal';
+import React, { useEffect, useState } from 'react';
+import { Alert, Button, FlatList, Image, Text, TouchableOpacity, View } from 'react-native';
+import FiltersModal from './components/FiltersModal.tsx';
+import { attractionsExamples } from '../api/fake/apiMock';
 import AttractionTile from '../../common/AttractionTile';
-import Swipeable from "react-native-gesture-handler/Swipeable";
+import Swipeable from 'react-native-gesture-handler/Swipeable';
 import { styles } from '../../Dashboard/styles';
-import { attractionSerchStyles } from './styles';
+import { attractionSearchStyles } from './styles';
 import { Attraction, Filters, NavigationProps } from '../types';
-import { fetchUsersAttractions } from './api/attractionsApi';
+import { fetchUserAttractions } from '../api/attractionsApi';
 
-const AttractionSearchScreen: React.FC = () => {
+const AttractionSearchScreen: React.FC<{ navigation: NavigationProps }> = ({ navigation }) => {
   const [loading, setLoading] = useState(true);
-  const [userAttractionsData, setUserAttractionsData] = useState<Attraction[]>(attractionsExamples);
+  const [userAttractions, setUserAttractions] = useState<Attraction[]>(attractionsExamples);
   const [attractions, setAttractions] = useState<Attraction[]>([]);
-  const [attractionDetails, setAttractionDetails] = useState<Attraction | {}>({});
   const [filterModalVisible, setFilterModalVisible] = useState(false);
-  const [attractionModalVisible, setAttractionModalVisible] = useState(false);
   const [filters, setFilters] = useState<Filters>({
-    category: "",
-    city: "",
-    country: "",
-    keyword: "",
-    radius: "10",
-    region: ""
+    category: '',
+    city: '',
+    country: '',
+    keyword: '',
+    radius: '10',
+    region: '',
   });
 
-  const [radiuses, setRadiuses] = useState<string[]>([]);
-  const navigation = useNavigation<NavigationProps>();
-
+  const [radii, setRadii] = useState<string[]>([]);
```

Comment:
what does mean `radii` ? I mean why there are 2 `i` letters at the end

Reply:
Radii are a collection of many radius.